pkg/reconciler/revision/resources/deploy.go

```diff
@@ -221,6 +221,13 @@ func MakeDeployment(rev *v1alpha1.Revision,
 		}
 	}
 
+	podSpec := makePodSpec(rev, loggingConfig, observabilityConfig, autoscalerConfig, deploymentConfig)
+	if _, ok := podTemplateAnnotations[istioIncludeInboundPortsAnnotation]; !ok {
+		queuePort := queueServingPort(rev.GetProtocol())
+		// Only intercept the serving port.
+		podTemplateAnnotations[istioIncludeInboundPortsAnnotation] = strconv.Itoa(int(queuePort.ContainerPort))
```

Comment:
Should this be Union of existing annotation ports plus this one?

Reply:
don't think so. I left the possibility to override in place just like all the others. If you want to override things, you're on your own and I think for now that's fine.